### PR TITLE
[All] Update to 1.1 SNAPSHOT

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: JetNews/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -42,7 +41,6 @@
     <string name="home_search">Search articles</string>
 
     <!-- Article Screen -->
-    <string name="article_published_in">Published in: %s</string>
```
Comment (astamato) [inline JetNews/app/src/main/res/values/strings.xml:45]: Were these unused or could you explain?

File: Jetchat/app/build.gradle [hunk 2]
```diff
@@ -78,7 +78,7 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerExtensionVersion '1.1.0-rc02' // For Kotlin 1.6.10. Revert with rc03.
+        kotlinCompilerExtensionVersion '1.1.0-SNAPSHOT' // For Kotlin 1.6.10. Revert with rc03.
```
Comment (astamato) [inline Jetchat/app/build.gradle:81]: Probably this comment should go when we update to remove snap?

**Review by astamato — APPROVED** [review]
2 minor obs, LGTM 💯

File: JetNews/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (florina-muntenescu) [reply JetNews/app/src/main/res/values/strings.xml:45]: yep, unused.

**Review by astamato — APPROVED** [review]

**Pull request merged**